spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/modal/ModalBottomSheet.md

```diff
@@ -0,0 +1,18 @@
+# Package com.adevinta.spark.components.bottomsheet.modal
+
+[BottomSheet](https://spark.adevinta.com/1186e1705/p/67d41e-bottom-sheet/b/02056b)
+A bottom sheet is a UI component commonly used in mobile applications to present additional content
+or options from the bottom of the screen.
+It is a modal component that slides up from the bottom of the screen and covers the entire screen.
+
+#### BottomSheet
```

Comment:
We should add more samples in the doc.
We can for ex add an example on how to display behind the handle like requested by design